www/index.html

```diff
@@ -466,6 +466,8 @@ <h3>Display settings</h3>
                                         <b>Select app theme</b> (content inversion is experimental):
                                     </label>
                                     <select class="form-control" id="appThemeSelect">
+                                        <option class="autoNotAvailable" value="auto_invert">Auto (generic)</option>
+                                        <option class="autoNotAvailable" value="auto_mwInvert">Auto (Wikimedia)*</option>
```

Comment:
This isn't a very intuitive class name. Why not just `class="auto"`, i.e. you just label the options as auto options?